back/engines/commercial/analysis/app/lib/analysis/auto_tagging_method/base.rb

```diff
@@ -39,5 +41,12 @@ def execute
     def update_progress(progress)
       task.update!(progress: progress)
     end
+
+    def deduct_locale(input)
+      input.author&.locale ||
+        input.title_multiloc&.keys&.first ||
+        input.body_multiloc&.keys&.first ||
+        AppConfiguration.instance.settings('core', 'locales').first
+    end
```

Comment:
Uhu, though in rare cases, multilingual inputs do make sense (option analysis, where gov posts these as different proposals). For now this should work :)

Reply:
Yes, I know; however, it's more of an exception rather than the norm. I think we could try to find ways to support this special use case while doing the right thing for normal ideas.

Also, in a way, it's a bit strange that we’re implementing ex-post language detection because we're aware that the keys of the multilocs aren't entirely dependable.